### PR TITLE
CI README-devs: make a recommendation on the form of overlay commit messages

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Comment by SkySkimmer** [comment]
cc @proux01 @ppedrot I do find it pretty annoying to have to dereference the github numbers when I see waiting overlay PRs that just say "adapt to coq/coq#XXX"

File: dev/ci/README-developers.md [hunk 1]
```diff
@@ -98,6 +98,14 @@ are the steps to manually prepare a patch:
 
 1. Fork the external project, create a new branch, push a commit adapting
    the project to your changes.
+
+   We recommend that the commit message mention your PR's number and a
+   short explanation of chat changed, eg
```
Comment (herbelin) [inline dev/ci/README-developers.md:103]:
```suggestion
   short explanation of what changed, eg
```

**Review by Zimmi48 — APPROVED** [review]
@coqbot merge now

**Comment by ejgallego** [comment]
Thanks @SkySkimmer , the hope at some point was that the `create-overlay` tooling would have a `commit` command that would take care of this in a more automated fashion.

**Pull request merged**